spec/unit/artifactserver_spec.rb

```diff
@@ -1,5 +1,7 @@
 require 'spec_helper'
 require 'cookbook-omnifetch/artifactserver'
+require 'zlib'
+require 'archive/tar/minitar'
```

Comment:
If mixlib-archive can create archives, it'd be cool to completely rip out minitar.